src/memory_tracker-inl.h

```diff
@@ -0,0 +1,197 @@
+#ifndef SRC_MEMORY_TRACKER_INL_H_
+#define SRC_MEMORY_TRACKER_INL_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "memory_tracker.h"
+
+namespace node {
+
+class MemoryRetainerNode : public v8::EmbedderGraph::Node {
+ public:
+  explicit inline MemoryRetainerNode(MemoryTracker* tracker,
+                                     const MemoryRetainer* retainer,
+                                     const char* name)
+     : retainer_(retainer) {
+    if (retainer_ != nullptr) {
+      v8::HandleScope handle_scope(tracker->isolate());
+      v8::Local<v8::Object> obj = retainer_->WrappedObject();
+      if (!obj.IsEmpty())
+        wrapper_node_ = tracker->graph()->V8Node(obj);
+
+      name_ = retainer_->MemoryInfoName();
+    }
+    if (name_.empty() && name != nullptr) {
+      name_ = name;
+    }
+  }
+
+  const char* Name() override { return name_.c_str(); }
```

Comment:
We may want to override `NamePrefix()` as well? Like using `Node /` so our nodes would appear with names `Node / Zlib` and they would be more distinguishable that way. V8 tend to use `system /` as a prefix for internal nodes like contexts and Chromium sometimes use `Window /` (although they seem to be just prepending the prefixes manually instead of using this API).

Reply:
Sounds good to me, done! :)